src/compoundSDK.js

```diff
@@ -7,7 +7,9 @@ import log from 'loglevel';
 
 export default class SDK {
 
-  async init() {
+  async init(apiKey) {
+    if(apiKey===undefined || apiKey==="")
```

Comment:
You can use if (!apiKey) as a very simple check. In the future we can verify more specifically.